CMakeLists.txt

```diff
@@ -214,19 +207,26 @@ IF(BUILD_PYTHON_INTERFACE)
     python/explog.hpp
     )
 
-  LIST(APPEND ${PROJECT_NAME}_PARSERS_HEADERS
-    parsers/python.hpp
-    )
-ENDIF(BUILD_PYTHON_INTERFACE)
-
-
-IF(HPP_FCL_FOUND AND BUILD_PYTHON_INTERFACE)
-  LIST(APPEND ${PROJECT_NAME}_PYTHON_HEADERS
+  IF(HPP_FCL_FOUND)
+    LIST(APPEND ${PROJECT_NAME}_PYTHON_HEADERS
       python/geometry-object.hpp
       python/geometry-model.hpp
       python/geometry-data.hpp
-    )
-ENDIF(HPP_FCL_FOUND AND BUILD_PYTHON_INTERFACE)
+      )
+  ENDIF(HPP_FCL_FOUND)
+
+ELSE(BUILD_PYTHON_INTERFACE)
+  SET(${PROJECT_NAME}_PYTHON_HEADERS "")
+  # LIST(APPEND ${PROJECT_NAME}_PARSERS_HEADERS
```

Comment:
The lines
` LIST(APPEND ${PROJECT_NAME}_PARSERS_HEADERS
     parsers/python.hpp
     )`
must appear in the block `IF(BUILD_PYTHON_INTERFACE) ... ELSE(BUILD_PYTHON_INTERFACE)` to handle the python loader. That's why the project does not compile with BUILD_PYTHON_INTERFACE sets to ON